wiki/Layout-component/universal-banner.md

```diff
@@ -0,0 +1,146 @@
+`Core` <!-- category start --><!-- category end -->
+
+A [[Universal banner]] is the only component allowed to show up above the [[Masthead]], and should be used exclusively for global and critical announcements (e.g. THINK global event, COVID 19 messages etc).
+
+![image](https://user-images.githubusercontent.com/3793636/140759323-201ebe94-7930-4201-addb-7de87504d330.png)
+
+> ![image](https://user-images.githubusercontent.com/3793636/117874180-493bdb80-b266-11eb-8945-dde0d95431d6.png)
+> There are a lot of similarities between this component and the [[Content group banner]]. Can these features be absorbed into that, or can [[Content group banner]] depend on this component or vice versa?
+
+<hr width="40%" />
+
+<!-- toc start open="true" depthStart="3" depthEnd="5" --><!-- toc end -->
+
+<details open="true">
+  <summary><strong>Dependencies</strong> (<!-- dependencyCount start --><!-- dependencyCount end -->)</summary><br />
+
+- [[Image]]
+- [[CTA]]
+- [[Lightbox media viewer]]
+- [[Markdown utility]]
+
+<br />
+</details>
+
+<!-- usedby start open="false" -->
+
+_{{usedby auto populates}}_
+
+<!-- usedby end -->
+
+<!-- backlinks start open="false" -->
+
+_{{backlinks auto populates}}_
+
+<!-- backlinks end -->
+
+<a name="resources"></a>
+
+<details open="true">
+  <summary><strong>Resources</strong></summary><br />
+
+- r1: [Box folder](https://ibm.ent.box.com/folder/142303458357)
+
+<br />
+</details>
+
+<hr width="40%" />
+
+<br />
+
+###### Logic
+
+|         | condition         | description                                                                                    |
+| :------ | :---------------- | :--------------------------------------------------------------------------------------------- |
+| `0.0.1` | content overflows | If the content extends beyond the component's fixed height than the content should be cut off. |
+| `0.0.2` | `@sm`             | Entire banner becomes clickable.                                                               |
+
+<br /><br />
+
+### 1. Media
+
+`optional` `component`
+
+An optional media element that can include either an [[Image]] or a [[Video]] at various sizes.
```

Comment:
I think we talked about this will only be Image, correct? We allow gifs but not real video files here.

Reply:
If we want the poster image, and the video to play in a lightbox I think we need video here.

Reply:
Looking back at our conversation around this. I see I misread your comment about the lightbox. Sorry about that, I can definitely remove this.